Src/MaintainableSelenium/MaintainableSelenium.API/Models/Category.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+
+namespace MaintainableSelenium.API.Models
+{
+    public class Category
+    {
+        public string Id { get; set; }
+        public string Name { get; set; }
+        public Browser Browser { get; set; }
```

Comment:
There could be many versions of the same browser (e.g. IE9, IE10). Maybe you should change field type to string?